docs/developer.md

```diff
@@ -304,6 +304,10 @@ The `IProtection` parent class provides a helper method for this in `calculate_l
 
 Most exchanges supported by CCXT should work out of the box.
 
+If you need to implement a specific exchange class, these are found in the `freqtrade/exchange` source folder.
+We recommend looking at existing exchange implementations to get an idea of what might be required.
+Implementing an exchange can be a lot of trial and error, so please bear this in mind. 
```

Comment:
And will require some dev experience?

Reply:
Definitely, but then I guess that's assumed when you're digging into freqtrade internals I suppose.